could not install on radxa rock5c

```bash
adxa@rock-5c:~/Softwares$ sudo dpkg -i /home/radxa/Softwares/aic8800-firmware_5.0+git20260123.5f7be68d-3_all.deb 
(正在读取数据库 ... 系统当前共安装有 233649 个文件和目录。)
准备解压 .../aic8800-firmware_5.0+git20260123.5f7be68d-3_all.deb  ...
正在解压 aic8800-firmware (5.0+git20260123.5f7be68d-3) 并覆盖 (5.0+git20260123.5f7be68d-1) ...
正在设置 aic8800-firmware (5.0+git20260123.5f7be68d-3) ...

radxa@rock-5c:~/Softwares$ sudo dpkg -i /home/radxa/Softwares/aic8800-usb-dkms_5.0+git20260123.5f7be68d-3_all.deb 
(正在读取数据库 ... 系统当前共安装有 233649 个文件和目录。)
准备解压 .../aic8800-usb-dkms_5.0+git20260123.5f7be68d-3_all.deb  ...
Deleting module aic8800-usb-5.0+git20260123.5f7be68d-1 completely from the DKMS tree.
正在解压 aic8800-usb-dkms (5.0+git20260123.5f7be68d-3) 并覆盖 (5.0+git20260123.5f7be68d-1) ...
正在设置 aic8800-usb-dkms (5.0+git20260123.5f7be68d-3) ...
Loading new aic8800-usb-5.0+git20260123.5f7be68d-3 DKMS files...
Building for 6.1.84-8-rk2410
Building initial module for 6.1.84-8-rk2410
Error! Bad return status for module build on kernel: 6.1.84-8-rk2410 (aarch64)
Consult /var/lib/dkms/aic8800-usb/5.0+git20260123.5f7be68d-3/build/make.log for more information.
dpkg: 处理软件包 aic8800-usb-dkms (--install)时出错： 已安装 aic8800-usb-dkms 软件包 post-installation 脚本 子进程返回错误状态 10
在处理时有错误发生： aic8800-usb-dkms

```

here is the log
```log
$ tail -n 30 /var/lib/dkms/aic8800-usb/5.0+git20260123.5f7be68d-3/build/make.log
  CC [M]  /var/lib/dkms/aic8800-usb/5.0+git20260123.5f7be68d-3/build/USB/driver_fw/drivers/aic8800/aic8800_fdrv/rwnx_radar.o
  CC [M]  /var/lib/dkms/aic8800-usb/5.0+git20260123.5f7be68d-3/build/USB/driver_fw/drivers/aic8800/aic8800_fdrv/rwnx_debugfs.o
  CC [M]  /var/lib/dkms/aic8800-usb/5.0+git20260123.5f7be68d-3/build/USB/driver_fw/drivers/aic8800/aic8800_fdrv/rwnx_fw_trace.o
  CC [M]  /var/lib/dkms/aic8800-usb/5.0+git20260123.5f7be68d-3/build/USB/driver_fw/drivers/aic8800/aic8800_fdrv/rwnx_testmode.o
  CC [M]  /var/lib/dkms/aic8800-usb/5.0+git20260123.5f7be68d-3/build/USB/driver_fw/drivers/aic8800/aic8800_fdrv/aicwf_tcp_ack.o
  CC [M]  /var/lib/dkms/aic8800-usb/5.0+git20260123.5f7be68d-3/build/USB/driver_fw/drivers/aic8800/aic8800_fdrv/usb_host.o
  CC [M]  /var/lib/dkms/aic8800-usb/5.0+git20260123.5f7be68d-3/build/USB/driver_fw/drivers/aic8800/aic8800_fdrv/aicwf_txrxif.o
  CC [M]  /var/lib/dkms/aic8800-usb/5.0+git20260123.5f7be68d-3/build/USB/driver_fw/drivers/aic8800/aic8800_fdrv/aicwf_usb.o
  CC [M]  /var/lib/dkms/aic8800-usb/5.0+git20260123.5f7be68d-3/build/USB/driver_fw/drivers/aic8800/aic8800_fdrv/aicwf_wext_linux.o
  LD [M]  /var/lib/dkms/aic8800-usb/5.0+git20260123.5f7be68d-3/build/USB/driver_fw/drivers/aic8800/aic8800_fdrv/aic8800_fdrv.o
make[1]: 警告: 子 make 中强制 -j8: 重置 jobserver 模式。  MODPOST /var/lib/dkms/aic8800-usb/5.0+git20260123.5f7be68d-3/build/USB/driver_fw/drivers/aic8800/Module.symvers
  CC [M]  /var/lib/dkms/aic8800-usb/5.0+git20260123.5f7be68d-3/build/USB/driver_fw/drivers/aic8800/aic8800_fdrv/aic8800_fdrv.mod.o
  CC [M]  /var/lib/dkms/aic8800-usb/5.0+git20260123.5f7be68d-3/build/USB/driver_fw/drivers/aic8800/aic_load_fw/aic_load_fw.mod.o
  LD [M]  /var/lib/dkms/aic8800-usb/5.0+git20260123.5f7be68d-3/build/USB/driver_fw/drivers/aic8800/aic8800_fdrv/aic8800_fdrv.ko
  LD [M]  /var/lib/dkms/aic8800-usb/5.0+git20260123.5f7be68d-3/build/USB/driver_fw/drivers/aic8800/aic_load_fw/aic_load_fw.ko
make: 离开目录“/usr/src/linux-headers-6.1.84-8-rk2410”
make: 进入目录“/usr/src/linux-headers-6.1.84-8-rk2410”
warning: the compiler differs from the one used to build the kernel
  The kernel was built by: aarch64-linux-gnu-gcc (Debian 12.2.0-14) 12.2.0
  You are using:           gcc (Debian 12.2.0-14+deb12u1) 12.2.0
  CC [M]  /var/lib/dkms/aic8800-usb/5.0+git20260123.5f7be68d-3/build/USB/driver_fw/drivers/aic_btusb/aic_btusb.o
/var/lib/dkms/aic8800-usb/5.0+git20260123.5f7be68d-3/build/USB/driver_fw/drivers/aic_btusb/aic_btusb.c: In function ‘bt_reboot_notify’:
/var/lib/dkms/aic8800-usb/5.0+git20260123.5f7be68d-3/build/USB/driver_fw/drivers/aic_btusb/aic_btusb.c:4948:9: error: unused variable ‘ret’ [-Werror=unused-variable]
 4948 |     int ret = 0;
      |         ^~~
cc1: all warnings being treated as errors
make[1]: *** [scripts/Makefile.build:250：/var/lib/dkms/aic8800-usb/5.0+git20260123.5f7be68d-3/build/USB/driver_fw/drivers/aic_btusb/aic_btusb.o] 错误 1
make: *** [Makefile:2010：/var/lib/dkms/aic8800-usb/5.0+git20260123.5f7be68d-3/build/USB/driver_fw/drivers/aic_btusb] 错误 2
make: 离开目录“/usr/src/linux-headers-6.1.84-8-rk2410”
```

--- a/debian/patches/fix-usb-build.patch
+++ b/debian/patches/fix-usb-build.patch
@@ -10,6 +10,18 @@ index d0970cb..656af12 100644
  
  RWNX_VERS_NUM := 6.4.3.0
  
+
+diff --git a/src/USB/driver_fw/drivers/aic_btusb/Makefile b/src/USB/driver_fw/drivers/aic_btusb/Makefile
+index 61ec3d0..0966f67 100644
+--- a/src/USB/driver_fw/drivers/aic_btusb/Makefile
++++ b/src/USB/driver_fw/drivers/aic_btusb/Makefile
+@@ -1,3 +1,5 @@
++EXTRA_CFLAGS += -Wno-unused-variable
++
+ MODULE_NAME = aic_btusb
+ 
+ CONFIG_AIC8800_BTUSB_SUPPORT = m
+
 diff --git a/src/USB/driver_fw/drivers/aic8800/aic8800_fdrv/rwnx_mod_params.c b/src/USB/driver_fw/drivers/aic8800/aic8800_fdrv/rwnx_mod_params.c
 index f78ca5f..3e7891f 100644
 --- a/src/USB/driver_fw/drivers/aic8800/aic8800_fdrv/rwnx_mod_params.c

